### ISSUE TITLE
Whitelist missing IP’s from shared-Skynet2-whitelist

### ISSUE THREAD

**Issue by wbartels** [Adamm00/IPSet_ASUS#21]
###### Steps To Reproduce Issue
After disabling "CDN Whitelisting" and Updating Malware Blacklist there are IP's missing from shared-Skynet2-whitelist:

```
104.18.45.190 comment "Shared-Whitelist: iplists.firehol.org"
192.168.1.2 comment "Shared-Whitelist: 192.168.1.2"
151.101.36.133 comment "Shared-Whitelist: 		raw.githubusercontent.com"
104.18.44.190 comment "Shared-Whitelist: iplists.firehol.org"
149.28.239.174 comment "Shared-Whitelist: ipdeny.com"
```

###### Data
Part of shared-Skynet-whitelist:
```
https://iplists.firehol.org/files/bds_atif.ipset
https://iplists.firehol.org/files/blocklist_de.ipset
https://iplists.firehol.org/files/greensnow.ipset
```

shared-Skynet2-whitelist:
```
ipdeny.com
                ipapi.co
                speedguide.net
                otx.alienvault.com
                raw.githubusercontent.com
                iplists.firehol.org
                astrill.com
                strongpath.net
                snbforums.com
                bin.entware.net
                nwsrv-ns1.asus.com
                nl.pool.ntp.org
                pool.ntp.org
                https://fwupdate.asuswrt-merlin.net
```

###### Suggestion
I think the tabs in  shared-Skynet2-whitelist are part of the problem.
Here a suggested to fix the Whitelist_Extra function:
```
Whitelist_Extra () {
                echo "ipdeny.com" > /jffs/shared-Skynet2-whitelist
                echo "ipapi.co" >> /jffs/shared-Skynet2-whitelist
                echo "speedguide.net" >> /jffs/shared-Skynet2-whitelist
                echo "otx.alienvault.com" >> /jffs/shared-Skynet2-whitelist
                echo "raw.githubusercontent.com" >> /jffs/shared-Skynet2-whitelist
                echo "iplists.firehol.org" >> /jffs/shared-Skynet2-whitelist
                echo "astrill.com" >> /jffs/shared-Skynet2-whitelist
                echo "strongpath.net" >> /jffs/shared-Skynet2-whitelist
                echo "snbforums.com" >> /jffs/shared-Skynet2-whitelist
                echo "bin.entware.net" >> /jffs/shared-Skynet2-whitelist
                echo "nwsrv-ns1.asus.com" >> /jffs/shared-Skynet2-whitelist
                echo "$(nvram get "ntp_server0")" >> /jffs/shared-Skynet2-whitelist
                echo "$(nvram get "ntp_server1")" >> /jffs/shared-Skynet2-whitelist
                echo "$(nvram get "firmware_server")" >> /jffs/shared-Skynet2-whitelist
}
```

**Comment by Adamm00** [comment]
Sorry I didn't reply to this earlier, totally forgot about it.

Fixed in #9de61c6140ca4abeaabe57548b7666cdcfca4d8f

**Comment by wbartels** [comment]
No problem, thanks for the fix!
